ansible/roles/debops.etherpad/defaults/main.yml

```diff
@@ -465,6 +483,41 @@ etherpad__mariadb__dependent_databases:
   - database: '{{ etherpad_database_name }}'
 
                                                                    # ]]]
+# .. envvar:: etherpad__postgresql__dependent_roles [[[
+#
+# User configuration for the debops.postgresql_ Ansible role.
+etherpad__postgresql__dependent_roles:
+
+  - name: '{{ etherpad_database_user }}'
+    password: '{{ etherpad_database_password }}'
+    flags: [ 'NOSUPERUSER', 'NOCREATEDB', 'LOGIN' ]
+
+                                                                   # ]]]
+# .. envvar:: etherpad__postgresql__dependent_groups [[[
+#
+# Group configuration for the debops.postgresql_ Ansible role.
+etherpad__postgresql__dependent_groups:
+
+                                                                   # ]]]
+# .. envvar:: etherpad__postgresql__dependent_databases [[[
+#
+# Database configuration for the debops.postgresql_ Ansible role.
+etherpad__postgresql__dependent_databases:
+
+  - name: '{{ etherpad_database_name }}'
+    port: '5432'
```

Comment:
I suppose this forces the usage in the `main` PostgreSQL cluster, but I guess it's fine for a initial support. We can update this later, when the PR is merged.

Reply:
I have a quick fix for this I was going to commit to my PR branch but I'm unsure what the desired process here is. The DebOps development process doc states `don't rewrite history in branches that are already pending as pull requests` but soon after that recommends a rebase during development as the upstream branch may have new commits. In this context I assume the rebase occurs before the initial feature branch push. I'm used to doing a rebase on PR branches as upstream changes because sometimes it seems necessary to allow it to be automatically merged and pass the checks. However, that seems to be against the desired process here. Have I understood this correctly? Am I to avoid a rebase once the PR has been created?

Reply:
My reasoning is this: having a large commit, unless it's something completely new like a new Ansible role, is bad for debugging and bisecting. With the PRs, if you create one, try and make small and sensible commits with any changes needed and push them to that PR's branch. You don't need to create new PRs (happened in the past), and you shouldn't try and `--force` new changes to existing branches, that messes with the reviewer's head when he sees the same commit names but there's a new code.

If there are no merge conflicts with the base branch, even if it moved on, you don't need to worry about rebasing. Merging code in a different place in the history is fine, as long as there are no conflicts.

if the PR cannot be merged cleanly, or the project moved so much forward that the changes don't make sense, then there's no other way around than rebasing the entire PR on the new commits in the base branch. That's acceptable as well, as long as you keep the commit history in that PR.

Since I added this PR on a separate branch to the DebOps monorepo, I think that the best case would be to close this PR, and create a new one against the `proposed/` branch. Then, when the time comes, that branch can be merged with the `master` branch without much issues, hopefully.

Keep in mind that I'm still new at this 'monorepo' thing and using multiple branches instead of separate repositories, we are developing best practices as we go. Any input is greatly appreciated. :-)

Reply:
Thanks, that clears up my confusion and it sounds reasonable so I don't have any recommendations at the moment.